addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1492,7 +1493,11 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
             if (ColorpickerWidget.isCSSColor(this._value)) {
                 this.colorPreviewEl.style.backgroundColor = this._value;
             } else if (!weUtils.isColorGradient(this._value)) {
-                this.colorPreviewEl.style.backgroundColor = `var(--we-cp-${this._value}`;
+                if(weUtils.EDITABLE_CSS_VARIABLES.includes(this._value) || compatibilityColorNames.includes(this._value)) {
+                    this.colorPreviewEl.style.backgroundColor = `var(--we-cp-${this._value}`;
+                } else {
+                    this.colorPreviewEl.classList.add(`${prefix}${this._value}`);
+                }
```

Comment:
I don't think you should use prefix here, but `bg-` but I am struggling understanding in which case this is actually useful but theoretically possible I guess? :grimacing:  
![image](https://user-images.githubusercontent.com/10338094/177362619-bf381cbe-ac34-48fe-ad49-2c412a8f5f6c.png)